package.json

```diff
@@ -43,7 +44,7 @@
 		"watch": "tsc -b -w",
 		"postinstall": "cd client && npm install && cd ../server && npm install && cd ..",
 		"test": "vscode-tmgrammar-test -s source.ion -g syntaxes/ion.tmLanguage.json -t \"client/test/highlight/**/*.ion\"",
-		"test:ui": "node ./client/out/test/runTest.js"
+		"ui-test": "npm run compile && extest setup-and-run client/out/ui-test/*.js -o .vscode/settings.json"
```

Comment:
This is the name for the build target used by GitHub Actions as well. See https://github.com/amzn/ion-vscode-plugin/blob/19c89e1028126338ced8cb4c504874d4cbd1770a/.github/workflows/build.yml#L33

If you are going to rename it, you need to update GitHub Action file and verify that the tests are running during a PR. 
This PR did **not** run the UI tests see https://github.com/amzn/ion-vscode-plugin/runs/1005448718?check_suite_focus=true#step:7:13

Reply:
I thought the line which I deleted was automatically added by the new test framework. I did not realize that it was here originally. I will add it back in the next commit.

Reply:
Sure. Add it back in and you might have to edit it to add the new tests that use `extest`